doc/book/hybrid.md

```diff
@@ -0,0 +1,119 @@
+# Encrypt and decrypt using hybrid cryptosystem
+
+Hybrid is an encryption mode that uses symmetric and public keys ciphers together.
+The idea is to take the advantages of the public key cryptography for sharing the
+keys and the speed of symmmetric encryption to encrypt the message.
+
+The hybrid mode is able to encrypt message for one or more receivers and can be
+used in multi user scenario, where you can limit the decryption only for some users.
+
+## How it works
+
+Suppose we have two users: *Alice* and *Bob*. *Alice* wants to send a message to *Bob*
+using an hybrid cryptosystem, she needs to:
+
+- Obtain *Bob*'s public key;
+- Generates a random session key (one-time pad);
+- Encrypts the message using a symmetric cipher with the previous session key;
+- Encrypts the session key using the *Bob*'s public key;
+- Send both of these encryptions to *Bob*.
```

Comment:
Rephrase to `Sends both the encrypted message and encrypted session key to *Bob*.`